CHANGELOG.md

```diff
@@ -4,6 +4,17 @@
 
 [1]: https://pypi.org/project/google-cloud-bigquery-datatransfer/#history
 
+## 2.0.0
+
+09-21-2020 08:36 PDT
+
+
+### Implementation Changes
+
+- Transition the library to microgenerator. ([#56](https://github.com/googleapis/python-bigquery-datatransfer/pull/56))
+  This is a **breaking change** that introduces several **method signature changes** and **drops support
```

Comment:
Link to UPGRADING guide?

Reply:
It's in the library docs ... at least I don't remember linking it in CHANGELOG?